src/components/proofEditor/proofEditor.tsx

```diff
@@ -3,7 +3,7 @@ import { checkProof } from "../../util/proofChecker";
 import styles from "./proofEditor.module.scss";
 
 export interface Props {
-
+	errorHere: string;

```

Comment:
```suggestion
```

This suggestion is just here so you can make the changes from GitHub and see if it works right away.